x-pack/plugins/fleet/server/services/agents/reassign.ts

```diff
@@ -108,94 +107,33 @@ export async function reassignAgents(
       }
     }
   } else if ('kuery' in options) {
-    return await processAgentsInBatches(
-      esClient,
-      {
-        kuery: options.kuery,
-        showInactive: options.showInactive ?? false,
-        batchSize: options.batchSize,
-      },
-      async (agents: Agent[], skipSuccess: boolean) =>
-        await reassignBatch(
-          soClient,
-          esClient,
-          newAgentPolicyId,
-          agents,
-          outgoingErrors,
-          undefined,
-          skipSuccess
-        )
-    );
+    const batchSize = options.batchSize ?? SO_SEARCH_LIMIT;
```

Comment:
for testing locally with less than 10k agents enrolled, set a lower number here:
```
const batchSize = 1000; // options.batchSize ?? SO_SEARCH_LIMIT;
```

or invoke from API:
```
POST kbn:/api/fleet/agents/bulk_reassign
{
  "agents": " fleet-agents.policy_id : (\"e57948b0-1d55-11ed-85fe-e34c31cf865b\" or \"26594150-1d56-11ed-85fe-e34c31cf865b\")",
  "policy_id": "e57948b0-1d55-11ed-85fe-e34c31cf865b",
  "batchSize": 1000
}
```